cognee-mcp/src/tools/search.py

```
@@ -0,0 +1,166 @@
+                elif (
+                    search_type.upper() == "GRAPH_COMPLETION"
+                    or search_type.upper() == "RAG_COMPLETION"
+                ):
+                    return str(search_results[0])
+                elif search_type.upper() == "CHUNKS":
+                    return str(search_results)
+                elif search_type.upper() == "INSIGHTS":
+                    results = retrieved_edges_to_string(search_results)
+                    return results
+                else:
+                    return str(search_results)
```

Pylint (4.0.2)

[refactor] Unnecessary "elif" after "return", remove the leading "el" from "elif"

(R1705)